Radio.Button 的 value 类型是不是和 Radio 不一样？

### Version
2.10.0

### Environment
typescript 2.3.2

### Reproduction link
[http://no.online.demo](http://no.online.demo)

### Steps to reproduce
``` ts
import {
  Form,
  Radio,
} from 'antd'
import * as React from 'react'

const RadioGroup = Radio.Group
const RadioButton = Radio.Button

export interface ISelectBooleanProps {
  form: any
}

class SelectBoolean extends React.Component<ISelectBooleanProps, any> {
  public render() {
    return (
      <Form>
        <RadioGroup>
          <RadioButton value={true}>true</RadioButton>
          <RadioButton value={false}>false</RadioButton>
        </RadioGroup>
      </Form>
    )
  }
}

export default Form.create()(SelectBoolean)

```

### What is expected?
这么写应该没有问题的吧，`Radio`的`value`是`any`哦，怎么`Radio.Button`不能传`boolean`呢？

### What is actually happening?
```
error TS2322: Type '{ value: false; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<RadioButton> & Readonly<{ children?: ReactNode; }>...'.
  Type '{ value: false; }' is not assignable to type 'Readonly<RadioButtonProps>'.
    Types of property 'value' are incompatible.
      Type 'false' is not assignable to type 'Key'.
```

<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/components/radio/radioButton.tsx
+++ b/components/radio/radioButton.tsx
@@ -1,13 +1,8 @@
 import React, { PropTypes } from 'react';
+import { AbstractCheckboxProps } from '../checkbox/Checkbox';
 import Radio from './radio';
 
-export interface RadioButtonProps {
-  value: string | number;
-  style?: React.CSSProperties;
-  disabled?: boolean;
-  checked?: boolean;
-  onChange?: (e: any) => any;
-}
+export interface RadioButtonProps extends AbstractCheckboxProps {}
 
 export default class RadioButton extends React.Component<RadioButtonProps, any> {
   static defaultProps = {

